### DISCUSSION TITLE
Doesn't connect to wallet after upgrading to 1.2.0

### DISCUSSION THREAD

**Discussion by AleksRaider** [Chia-Network/chia-blockchain#7216] [Support]
Does not connect to the wallet after upgrading to 1.2.0 under Windows 7, tried to delete everything, nothing helps, just twists "Connect to wallet"

**Comment by ALEX7634863443** [comment]
I've the same problem under Windows 7. 
![2021-07-08_15-26-50](https://user-images.githubusercontent.com/87128382/124914449-06457c80-dff9-11eb-99c1-f331bf2d529a.png)

Endless error "Connecting to wallet"

Try to delete 1.2.0 and install v. 1.1.7 and all is ok, then try to update to 1.2.0 and the problem say Hello once again.

**Reply by ALEX7634863443** [reply]
Let's make Chia blockchain better together ;)

**Comment by AleksRaider** [comment]
I have the same situation and I also know 3 people who have the same. When installing 1.1.7 everything works

**Reply by Jones1982** [reply]
Idem.

**Comment by loppefaaret** [comment]
are you all on windows 7 ? windows 7 is no longer supported!

**Reply by Jones1982** [reply]
Yes, Windows7 Sp1 64bit

**Reply by loppefaaret** [reply]
we are running python 3.9 now under windows, and windows 7 are missing a .dll file that does not official allow it to run python 3.9 - so unless you can find a way around that - on your own responsibility - that is it for windows 7 and Chia, you are stuck on 1.1.7 for as long as it works

**Reply by Jones1982** [reply]
talk about the famous file "api-ms-win-core-path-l1-1-0.dll"?

**Reply by loppefaaret** [reply]
just be careful if you try and download system files from a 3rd party!

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
